src/v3/src/hooks/useOnSubmit.ts

```diff
@@ -51,6 +53,15 @@ export const useOnSubmit = (): (options: OnSubmitHandlerOptions) => Promise<void
       stepToRender,
     } = options;
 
+    let { stateHandle } = currTransaction!.context;
+
+    if (step === 'cancel') {
+      authClient?.transactionManager.clear({ clearIdxResponse: false });
```

Comment:
I added this line to match what v2 is currently doing when it encounters a `cancel` action: https://github.com/okta/okta-signin-widget/blob/master/src/v2/controllers/FormController.ts#L165-L177